app/Module/CensusAssistantModule.php

```diff
@@ -299,12 +288,39 @@ public static function formatCensusNote(Note $note) {
 			$data      = Filter::escapeHtml($match[4]);
 			$postamble = Filter::escapeHtml($match[5]);
 
+			// Get the column headers for the census to which this note refers
+			// assumes a soundex match between the fact place name and the specific
+			// census censusDate() function
+			$xref      = $note->getXref();
+			$firstIndi = $note->linkedIndividuals('NOTE')[0];
```

Comment:
Dereferencing arrays like this won't work in PHP 5.3.  You need a temp variable.

What if there are no linked individuals?